draft-ietf-teep-protocol.md

```diff
@@ -1520,13 +1520,29 @@ the selected TEEP cipher suite MUST be used in both directions.
 ## EATs and SUIT Reports {#eat-suit-ciphersuite}
 
 TEEP uses COSE for confidentiality of EATs and SUIT Reports sent by a TEEP Agent.
-A TAM MUST support both of the cipher suites defined below, which are defined to
-be consistent with profiles listed in {{I-D.moran-suit-mti}}.  A TEEP Agent MUST
-support at least one of the two but can choose which one.  For example, a TEEP
-Agent might choose a given cipher suite if it has hardware support for it.
-A TAM or TEEP Agent MAY also support other algorithms in the COSE Algorithms
-registry in addition to the mandatory ones listed below.  It MAY also support use
-with COSE_Encrypt or other COSE types in additional cipher suites.
+EATs are typically signed by the attester, which is component different from the
+TEEP Agent. The TEEP Agent then obtains the signed EAT and encrypts it with the TAM
```

Comment:
I don't think it's safe to say that the attester is different from the TEEP Agent.   If the TEEP Agent is attesting to the TAM, then the TEEP Agent is the top layer of the attester, so is part of it.

Reply:
My point is that the software/hardware component doing the attestation is different from the software implementation of the TEEP Agent. As such, the encryption functionality will be applied after the attester-component has created the EAT and signed it.

I am happy to soften the sentence by saying "... which may be a software component different from the
TEEP Agent.".